config/locales/administrate.pt-BR.yml

```diff
@@ -22,8 +22,8 @@ pt-BR:
         more: "Exibindo %{count} de %{total_count}"
         none: Nenhum
     form:
-      error: error
-      errors: "%{pluralized_errors} prohibited this %{resource_name} from being saved:"
+      error: erro
```

Comment:
Is dropping the `r` here deliberate?

Reply:
yeah, `erro` is the correct translation for `error`

Reply:
Oh huh! Cool, thanks!